test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -94,6 +95,77 @@ public void It_publishes_self_contained_apps_to_the_publish_folder_and_the_app_s
                 .HaveStdOutContaining("Hello World!");
         }
 
+        //Note: Pre Netcoreapp2.0 stanalone activation uses renamed dotnet.exe
+        //      While Post 2.0 we are shifting to using apphost.exe, so both publish needs to be validated
+        [Fact]
+        public void Publish_standalone_post_netcoreapp2_app_and_it_should_run()
+        {
+            var targetFramework = "netcoreapp2.0";
+            var rid = EnvironmentInfo.GetCompatibleRid(targetFramework);
+
+
+            TestProject referencerProject = new TestProject()
+            {
+                Name = "Hello",
+                IsSdkProject = true,
+                TargetFrameworks = targetFramework,
+                RuntimeFrameworkVersion = RepoInfo.NetCoreApp20Version,
+                // Need to use a self-contained app for now because we don't use a CLI that has a "2.0" shared framework
```

Comment:
this comment doesn't belong.  We use a self-contained app because thatis what we are testing.